.github/workflows/check.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
```

Comment:
I suggest using fixed commits as action versions (For all three changes). It Provides Better security.